releasenotes/v1.10.0-beta.0.md

```diff
@@ -0,0 +1,170 @@
+[#2360](https://github.com/metal3-io/cluster-api-provider-metal3/issues/2360)
+[#2358](https://github.com/metal3-io/cluster-api-provider-metal3/issues/2358)
+[#2347](https://github.com/metal3-io/cluster-api-provider-metal3/issues/2347)
+[#2333](https://github.com/metal3-io/cluster-api-provider-metal3/issues/2333)
+[#2342](https://github.com/metal3-io/cluster-api-provider-metal3/issues/2342)
+[#2109](https://github.com/metal3-io/cluster-api-provider-metal3/issues/2109)
+[#2321](https://github.com/metal3-io/cluster-api-provider-metal3/issues/2321)
+[#2317](https://github.com/metal3-io/cluster-api-provider-metal3/issues/2371)
+[#2318](https://github.com/metal3-io/cluster-api-provider-metal3/issues/2318)
+[#2308](https://github.com/metal3-io/cluster-api-provider-metal3/issues/2308)
+[#2312](https://github.com/metal3-io/cluster-api-provider-metal3/issues/2312)
+[#2298](https://github.com/metal3-io/cluster-api-provider-metal3/issues/2298)
+[#2307](https://github.com/metal3-io/cluster-api-provider-metal3/issues/2307)
+[#2300](https://github.com/metal3-io/cluster-api-provider-metal3/issues/2300)
+[#2302](https://github.com/metal3-io/cluster-api-provider-metal3/issues/2302)
+[#2289](https://github.com/metal3-io/cluster-api-provider-metal3/issues/2289)
+[#2259](https://github.com/metal3-io/cluster-api-provider-metal3/issues/2259)
+[#2276](https://github.com/metal3-io/cluster-api-provider-metal3/issues/2276)
+[#2277](https://github.com/metal3-io/cluster-api-provider-metal3/issues/2277)
+[#2266](https://github.com/metal3-io/cluster-api-provider-metal3/issues/2266)
+[#2240](https://github.com/metal3-io/cluster-api-provider-metal3/issues/2240)
+[#2242](https://github.com/metal3-io/cluster-api-provider-metal3/issues/2242)
+[#2246](https://github.com/metal3-io/cluster-api-provider-metal3/issues/2246)
+[#2244](https://github.com/metal3-io/cluster-api-provider-metal3/issues/2244)
+[#2203](https://github.com/metal3-io/cluster-api-provider-metal3/issues/2203)
+[#2235](https://github.com/metal3-io/cluster-api-provider-metal3/issues/2235)
+[#2212](https://github.com/metal3-io/cluster-api-provider-metal3/issues/2212)
+[#2218](https://github.com/metal3-io/cluster-api-provider-metal3/issues/2218)
+[#2216](https://github.com/metal3-io/cluster-api-provider-metal3/issues/2216)
+[#2202](https://github.com/metal3-io/cluster-api-provider-metal3/issues/2202)
+[#2201](https://github.com/metal3-io/cluster-api-provider-metal3/issues/2201)
+[#2194](https://github.com/metal3-io/cluster-api-provider-metal3/issues/2194)
+[#2183](https://github.com/metal3-io/cluster-api-provider-metal3/issues/2183)
+[#2172](https://github.com/metal3-io/cluster-api-provider-metal3/issues/2172)
+[#2178](https://github.com/metal3-io/cluster-api-provider-metal3/issues/2178)
+[#2164](https://github.com/metal3-io/cluster-api-provider-metal3/issues/2164)
+[#2165](https://github.com/metal3-io/cluster-api-provider-metal3/issues/2165)
```

Comment:
Why are these all just the issue number and link? Not a big deal for the beta but later we need proper descriptions like for the other sections

Reply:
I just done it based on prior art, AFAIK the revert/superseded section is just a list everywhere.